include/git2/common.h

```diff
@@ -305,6 +306,9 @@ typedef enum {
  *		>
  *		> - `ciphers` is the list of ciphers that are eanbled.
  *
+ *	* opts(GIT_OPT_ENABLE_OFS_DELTA, int enabled)
+ *
+ *		> Enable or disable ofs_delta capability.
```

Comment:
I would add some more documentation here.  Here's an example (though I'm not tied to this):

> Enable or disable creating "offset deltas" when creating packfiles.  Offset deltas store a delta base location as an offset into the packfile from the current location, which provides a shorter encoding and thus smaller resultant packfiles.  Packfiles containing offset deltas can still be read.  This defaults to enabled.